### PR TITLE
feat: UI for admin/sales/marketer page

### LINKED ISSUE #243: Add UI for admin/sales/marketer
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

screenshot from 2017-06-17 10-29-22

### PR DESCRIPTION
**Pull request by harshitagupta30**
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This PR adds the UI for the admin/sales/marketer page/

Fixes #243 

Deployment Link: https://open-event-frontend-harshita.herokuapp.com/admin/sales/marketer

@niranjan94 @CosmicCoder96 @geekyd @sumedh123 @utkarshnath @anu-007 Please Review :+1:

### CODE REVIEW

File: app/templates/admin/sales/marketer.hbs [hunk 1]
```diff
@@ -1 +1,38 @@
...
+          {{#each model as |entry|}}
+            <tr>
+              <td>{{entry.marketers}}</td>
+              <td class="right aligned">{{entry.completedTickets}}</td>
+              <td class="right aligned">{{number-format entry.completedSales}}</td>
+              <td class="right aligned">{{number-format entry.completedDiscounts}}</td>
+            </tr>
+          {{/each}}
+          <tr>
+            <td>{{t 'Total'}}</td>
```
Comment (geekyd) [inline app/templates/admin/sales/marketer.hbs]: Please remove this row from here & add this to `tfoot`

Comment (harshitagupta30) [inline app/templates/admin/sales/marketer.hbs]: updated! please review again. Thanks

File: app/templates/admin/sales/marketer.hbs [hunk 2]
```diff
@@ -1 +1,40 @@
...
+              <td class="right aligned">{{entry.completedTickets}}</td>
+              <td class="right aligned">{{number-format entry.completedSales}}</td>
+              <td class="right aligned">{{number-format entry.completedDiscounts}}</td>
+            </tr>
+          {{/each}}
+        </tbody>
+        <tfoot>
+          <tr>
+            <th>{{t 'Total'}}</th>
+            <th class="right aligned">0</th>
```
Comment (geekyd) [inline app/templates/admin/sales/marketer.hbs]: These values should be calculated using the data in the model. Please create an object that holds all the totals.

File: app/templates/admin/sales/marketer.hbs [hunk 3]
```diff
@@ -1 +1,40 @@
...
+      <table class="ui stackable structured celled table">
+        <thead>
+          <tr>
+            <th rowspan="2">{{t 'Marketers'}}</th>
+            <th colspan="3" class="ui green inverted segment center aligned">{{t 'Completed Orders '}}</th>
+          </tr>
+          <tr>
+            <th class="right aligned">{{t 'Tickets'}}</th>
+            <th class="right aligned">{{t 'Sales (discounted)'}}</th>
+            <th class="right aligned">{{t 'Discounted amount'}}</th>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/marketer.hbs]: Make the 'a' in amount capital please :)

File: app/templates/admin/sales/marketer.hbs (hunk 2 above)
Comment (harshitagupta30) [inline app/templates/admin/sales/marketer.hbs]: @geekyd Thanks! updated :+1: Please review again.

Comment (geekyd) [inline app/templates/admin/sales/marketer.hbs]: @harshitagupta30 Is the deployment link updated?

Comment (harshitagupta30) [inline app/templates/admin/sales/marketer.hbs]: @geekyd Yes it's updated.
Link : https://open-event-frontend-harshita.herokuapp.com/admin/sales/marketer

**Pull request merged**